pkg/controller/storage/account/account.go

```diff
@@ -67,17 +65,20 @@ type Reconciler struct {
 	managed.ReferenceResolver
 	managed.Initializer
 
+	poll time.Duration
+
 	log logging.Logger
 }
 
 // Setup adds a controller that reconciles Accounts.
-func Setup(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter) error {
+func Setup(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter, poll time.Duration) error {
```

Comment:
The passing of the poll interval through the various nested syncers / updaters in the storage controllers is not super clean, but I opted to go for path of least resistance here since these controllers need to be refactored to use the managed reconciler anyway.